examples/linear_model/plot_sgd_penalties.py

```diff
@@ -5,9 +5,7 @@
 
 Plot the contours of the three penalties.
 
-All of the above are supported by
-:class:`sklearn.linear_model.stochastic_gradient`.
-
+All of the above are supported by :ref:`sgd`.
```

Comment:
because it is both for regressor and classifier. That also seems to be from one of my PRs, right?